app/models/user.rb

```diff
@@ -139,6 +139,11 @@ def should_see_beta?(feature = nil)
     in_beta_program? && active_beta_user? && feature
   end
 
+  def can_see_beta?(feature = nil)
+    feature = feature ? ENV[feature] == "true" : true
```

Comment:
This is hard for me to follow — what type is `feature` expected to be? Like, a String with a feature name? And then line 143 changes its type from a String to a Boolean?